gui/default/syncthing/settings/settingsModalView.html

```diff
@@ -159,6 +159,165 @@
               </div>
             </div>
           </div>
+
+          <!-- WebAuthn settings -->
+          <div class="form-group">
+
+            <!-- WebAuthn credentials -->
+            <div class="table-responsive webauthn-credentials">
+              <table class="table-striped table">
```

Comment:
Maybe we could address the long table rows issue similar to how the main view does it for folders and devices.  When the width gets smaller, the cells wrap within a table row.  IIRC this is mainly affected by the `display: block;` and the width settings on `<th>` and `<td>` in the respective media queries in `overrides.css`.

Similar thing could be done here, to make kind of a card layout. Keep the credential ID top, left aligned, ellipsized.  Make the other columns' cells wrap below that, right aligned, including the action buttons.

We can further discuss this in detail in another meta PR, or postpone it until after this PR.  I'd be fine with having suboptimal (non-)responsive design as a first step, and improve later.

Reply:
Here's an experiment in this direction, please let me know what you think!

- https://github.com/emlun/syncthing/pull/12

Reply:
Fixed in https://github.com/emlun/syncthing/pull/12.